tests/test_regr.py

```diff
@@ -51,9 +51,8 @@ def disable():
 
 @pytest.fixture
 def finalize_linter(linter: PyLinter) -> Iterator[PyLinter]:
-    """call reporter.finalize() to cleanup
-    pending messages if a test finished badly
-    """
+    """Call reporter.finalize() to clean up pending messages if a test
+    finished badly."""
```

Comment:
```suggestion
    finished badly.
    """
```

This is a PEP8 recommendation.